src/include/pbs_internal.h

```diff
@@ -548,6 +548,77 @@ extern char *convert_time(char *);
 extern struct batch_status *bs_isort(struct batch_status *bs,
 	int (*cmp_func)(struct batch_status*, struct batch_status *));
 
+extern int __pbs_asyrunjob(int, char *, char *, char *);
```

Comment:
Instead of overloading pbs_internal.h, how about putting these declarations in a new header file, something like "ifl_internal.h" ? I think it will make things cleaner, but it's up to you.